ps-pxc-dist/Jenkinsfile.groovy

```diff
@@ -0,0 +1,104 @@
+        )
+        string(
+            description: 'Must be in form $DESTINATION/*****/$releaseXXX/$revision',
+            name: 'QPRESS_PATH'
+        )
+        string(
+            description: 'Must be in form $DESTINATION/*****/$releaseXXX/$revision',
+            name: 'PERCONA_HAPROXY_PATH'
+        )
+        string(
+            description: 'Must be in form $DESTINATION/*****/$releaseXXX/$revision',
+            name: 'PERCONA_PROXYSQL_PATH'
+        )
+        string(
+            description: 'Must be in form $DESTINATION/*****/$releaseXXX/$revision',
+            name: 'PERCONA_TOOLKIT_PATH'
+        )
+        string(
+            description: 'Must be in form $DESTINATION/*****/$releaseXXX/$revision',
+            name: 'PERCONA_XTRABACKUP_PATH'
+        )
+        string(
+            description: 'Must be in form $DESTINATION/*****/$releaseXXX/$revision',
+            name: 'PERCONA_PXC_PATH'
+        )
+        string(
+            description: 'Must be in form $DESTINATION/*****/$releaseXXX/$revision',
+            name: 'PERCONA_PS_PATH'
+        )
+        string(
+            description: 'Must be in form $DESTINATION/*****/$releaseXXX/$revision',
+            name: 'PERCONA_MYSQL_SHELL_PATH'
+        )
+        string(
+            description: 'Must be in form $DESTINATION/*****/$releaseXXX/$revision',
+            name: 'PERCONA_ORCHESTRATOR_PATH'
+        )
+        choice(
+            choices: 'PDPS\nPDPXC',
+            description: 'Repository push to',
+            name: 'REPOSITORY'
+        )
+        string(
+            description: 'Version of repository push to',
+            name: 'REPOSITORY_VERSION'
+        )
+        choice(
+            choices: 'TESTING\nRELEASE\nEXPERIMENTAL\nLABORATORY',
+            description: 'Separate repository to push to',
+            name: 'COMPONENT'
+        )
+        booleanParam(
+            defaultValue: false,
+            description: "Remove lockfile after unsuccessful push for DEB",
+            name: 'REMOVE_LOCKFILE'
+        )
+        booleanParam(
+            defaultValue: false,
+            description: "Check to remove sources and binary version if equals pushing",
+            name: 'REMOVE_BEFORE_PUSH'
+        ) 
+    }
+    options {
+        skipDefaultCheckout()
+        disableConcurrentBuilds()
+    }
+
+    stages {
+        stage('push-to-rpm-repository') {
+            steps {
+                sh '''
+                    wget https://raw.githubusercontent.com/Sudokamikaze/jenkins-pipelines/DISTMYSQL-36/ps-pxc-dist/rpm_release.sh
+                    bash -xe rpm_release.sh
+                '''
+            }
+        }
+        stage('push-to-deb-repository') {
+            steps {
+                sh '''
+                    wget https://raw.githubusercontent.com/Sudokamikaze/jenkins-pipelines/DISTMYSQL-36/ps-pxc-dist/apt_release.sh
+                    bash -xe apt_release.sh
+                '''
+            }
+        }
+    }
+
```

Comment:
please add sync repos stage and clean_cdn_cache

Reply:
Added few seconds ago :)